### DISCUSSION TITLE
Migration guide for "Webpacker to Shakapacker v6 / v7"

### DISCUSSION THREAD

**Discussion by coingates** [shakacode/shakapacker#344] [Ideas]
Is there any migration guide for projects using `react-on-rails` and `webpacker` to fully utilize `shakapacker` (either v6 or v7) with some of the post-migration code adjustment scenarios?

Like `asset_pack_path` base path changing from `media/...` to `static/...`? And other such cases for a full migration.

I've search the website (via Algolia powered search input) for keyword `migration` and there's not many results. None of the results covered this scenario.

Is there any chance for a full migration guide anytime soon?

Thanks for your input! I'll be in touch for your questions and comments.

**Comment by ahangarha** [comment]
Shakapacker has a guide for upgrading [from Webpacker to Shakapacker 6](../tree/master/docs/v6_upgrade.md) and from [Shakapacker 6 to 7](../tree/master/docs/v7_upgrade.md)

If your main concern is about react_on_rails-specific issues, please [open a discussion](https://github.com/shakacode/react_on_rails/discussions/new/choose) on that project if documentation is insufficient.

Besides this, aim for upgrading to Shakapacker 7 for better future support and new features.
